packages/nextjs/components/scaffold-eth/toast.tsx

```diff
@@ -18,6 +18,36 @@ function Toast(message: string, alertClass: toastClassType) {
   });
 }
 
+// TODO Remove this in future
+/**
+ * Custom Txn Toast
+ */
+function TxToast(blockExplorerLink: string) {
```

Comment:
Really sorry for adding this here in this file, was not able to find a better place at that moment 😅, please let me know if I should move it somewhere else

Reply:
We definitely need to think better about how we use the toasts. Rn it's a bit messy, using react-hot-toast directly & our custom version on top of it at the same time.

Can your create an issue for it? We can discuss there. There are a few options.

But this is great, we've been using toast for a while, and now we understand better our needs.

Reply:
Yes, I updated the custom Txn toast, for now, will create an issues for Toasts